cmd/ocm-backplane/login/login.go

```diff
@@ -183,6 +185,13 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	}
 
 	logger.Debugf("Using backplane URL: %s\n", bpURL)
+	backplanehost, err := getBackplaneHost(bpURL)
+	if err != nil {
+		logger.Errorf(err.Error())
+	} else {
+		logger.Debugf("Backplane URL resolves to %s \n", backplanehost)
+		logger.Debugf("Please refer to https://source.redhat.com/groups/public/sre/wiki/backplane_troubleshooting to figure out the associated backplane server")
```

Comment:
Not sure if print the internal doc here is good, I would prefer to remove this.

Reply:
+1 please purge this

Reply:
```
	if err != nil {
		logger.Warn(err.Error())

	} else {
		logger.Debugf("Backplane URL resolves to %s \n", backplaneResolution)
		logger.Debugf("Please refer to your design doc/troubleshooting doc to figure out the associated backplane server")
	}
```
In between complete purging and giving the link to the document, I am suggesting best of both worlds.